projects/1.20/assets/butchers-delight-foods/butchersdelightfoods/lang/zh_cn.json

```
@@ -0,0 +1,34 @@
+{
+  "item.butchersdelightfoods.sheeploin": "羊腰",
+  "itemGroup.tabbutcher_foods": "屠夫乐事的肉",
+  "item_group.butchersdelightfoods.butcher_foods": "屠夫乐事的肉",
+  "item.butchersdelightfoods.beeftenderloin": "生牛里脊",
+  "item.butchersdelightfoods.goat_loin": "山羊腰",
+  "item.butchersdelightfoods.cookedsheepshank": "熟羊腿",
+  "item.butchersdelightfoods.coockedbeeftenderloin": "熟牛里脊",
+  "item.butchersdelightfoods.porkloin": "猪腰肉",
+  "item.butchersdelightfoods.cooked_llama_loin": "熟羊驼腰",
+  "item.butchersdelightfoods.cooked_goat_loin": "熟山羊腰",
+  "item.butchersdelightfoods.cooked_llama_leg": "熟羊驼腿",
+  "item.butchersdelightfoods.sheepshank": "生羊腿",
+  "item.butchersdelightfoods.cookedbeefribs": "熟牛肋排",
+  "item.butchersdelightfoods.ham": "生火腿",
+  "item.butchersdelightfoods.cookedporkloin": "熟猪里脊",
+  "item.butchersdelightfoods.llamma_ribs": "羊驼肋排",
+  "item.butchersdelightfoods.beefribs": "生牛肋排",
+  "item.butchersdelightfoods.goat_shank": "生山羊腿",
+  "item.butchersdelightfoods.cooked_goat_rack": "熟山羊肋排",
+  "item.butchersdelightfoods.llama_loin": "羊驼腰",
+  "item.butchersdelightfoods.legcow": "生牛腿",
+  "item.butchersdelightfoods.sheeprack": "羊肋排",
+  "item.butchersdelightfoods.cooked_sheeploin": "熟羊腰",
+  "item.butchersdelightfoods.goatrack": "山羊肋排",
```

| **审阅级别** |
|:--------------:|
|⚪ **轻微** / 不阻止合并但需三思|

原文：**Raw Goat Rack**
```suggestion
  "item.butchersdelightfoods.goatrack": "山羊排",
```